README.md

```diff
@@ -33,6 +39,36 @@ module "iam_roles" {
     "ROLE-DEV" = "arn:aws:iam::*:policy/perm-boundaries/default"
   }
 
+  # List of users to manage
```

Comment:
Given that TF 0.14 introduced this _experimental_ feature, I would suggest to finally leverage it: [module_variable_optional_attrs ](https://github.com/hashicorp/terraform/blob/v0.14.0/CHANGELOG.md) :)
It's going to make everything much cleaner.